aioshelly/wsrpc.py

```diff
@@ -151,7 +156,10 @@ def _handle_frame(self, frame):
         else:
             _LOGGER.warning("Invalid frame: %s", frame)
 
-    async def _rx_msgs(self):
+    async def _rx_msgs(self) -> None:
+        if not self._client:
+            return
```

Comment:
This can't happen so it is better to assert here and not just return
```suggestion
        assert self._client
```